test/unit/atomic/src/AtomicTest.cpp

```diff
@@ -782,10 +782,6 @@ using TestAccs = alpaka::test::EnabledAccs<alpaka::DimInt<1u>, std::size_t>;
 TEMPLATE_LIST_TEST_CASE("atomicOperationsWorking", "[atomic]", TestAccs)
 {
     using Acc = TestType;
-    TestAtomicOperations<Acc, unsigned char>::testAtomicOperations();
-    TestAtomicOperations<Acc, char>::testAtomicOperations();
-    TestAtomicOperations<Acc, unsigned short>::testAtomicOperations();
-    TestAtomicOperations<Acc, short>::testAtomicOperations();
```

Comment:
I think this is too harsh, as now we are no longer testing 8 and 16 bit atomics on CPUs, which support these. Please also synchronize with @tonydp03, who is also working on this in #1431.

Reply:
This is not supposed to do redundant work. This is just a temporary measure following one path of the currently inconsistent handling of atomic tests.  #1431 is expected to fix this. I would like to have this PR merged first so that #1431 can the requirements herein into account.

Reply:
Could you plese then comment the tests out and add a comment that those should be reenabled later for the targets supporting them? I want to avoid getting these tests lost when #1431 rebases later (whenever that is).

Reply:
Has this been resolved? @jkelling @bernhardmgruber

Reply:
Ah, sorry for the noise. I should have read further...

Reply:
Yes, these lines are in, but commented, with a note for the atomic test rework.
https://github.com/alpaka-group/alpaka/pull/1308/files#diff-97505ba324687f46490362096f3ee6b11b77b812ca5e918fa54e956304865c81